### PR TITLE
feat: annotate node load by node name

### PR DESCRIPTION
**Pull request by wuxs**
#18 
#8

### CODE REVIEW

File: pkg/controller/annotator/node.go [hunk 1]
```diff
@@ -100,8 +100,14 @@ func (n *nodeController) syncNode(key string) (bool, error) {
 
 func annotateNodeLoad(promClient prom.PromClient, kubeClient clientset.Interface, node *v1.Node, key string) error {
 	value, err := promClient.QueryByNodeIP(key, getNodeInternalIP(node))
```
Comment (imneov) [inline pkg/controller/annotator/node.go:102]: Would be better like: 
```
         value, err := promClient.QueryByNodeIP(key, getNodeInternalIP(node))
	if err == nil && len(value) > 0 {
                 return patchNodeAnnotation(kubeClient, node, key, value)
        }
        value, err = promClient.QueryByNodeName(key, getNodeName(node))
	if err == nil && len(value) > 0 {
                 return patchNodeAnnotation(kubeClient, node, key, value)
        }
        return fmt.Errorf("failed to get data %s{%s=%s}: %v", key, node.Name, value, err)
```

File: pkg/controller/prometheus/prometheus.go [hunk 2]
```diff
@@ -21,6 +21,8 @@ const (
 type PromClient interface {
 	// QueryByNodeIP queries data by node IP.
 	QueryByNodeIP(string, string) (string, error)
+	// QueryByNodeIP queries data by node IP.
```
Comment (autumn0207) [inline pkg/controller/prometheus/prometheus.go:24]: Change `node IP` to `node Name`

File: pkg/controller/prometheus/prometheus.go [hunk 3]
```diff
@@ -64,6 +66,25 @@ func (p *promClient) QueryByNodeIP(metricName, ip string) (string, error) {
...
+	klog.V(4).Infof("Try to query %s by node name[%s]", metricName, name)
+
+	querySelector := fmt.Sprintf("%s{instance=~\"%s\"} /100", metricName, name)
+
+	result, err := p.query(querySelector)
+	if result != "" && err == nil {
+		return result, nil
+	}
+
+	querySelector = fmt.Sprintf("%s{instance=~\"%s:.+\"} /100", metricName, name)
```
Comment (autumn0207) [inline pkg/controller/prometheus/prometheus.go:79]: No fuzzy matching is required for `node Name`, and this part can be removed.

**Review by xieydd — CHANGES_REQUESTED** [review]
can we fix via the comment from @autumn0207

**Comment by wuxs** [comment]
> can we fix via the comment from @autumn0207

ok

**Review by xieydd — APPROVED** [review]
/lgtm

**Comment by xieydd** [comment]
Thanks for your controbution @wuxs

**Pull request merged**